src/content/doc-surrealdb/deployment/azure.mdx

```diff
@@ -54,20 +54,20 @@ This tutorial is intended for production workloads using the <code>standard</cod
 
 1. Log in to Azure and get the current subscription if you have multiple subscriptions.
 
-```bash title="Log in to Azure and list Azure subscriptions"
```

Comment:
Hey @Dhghomon did you mean to remove the titles? It's usually helpful to leave these in for screen readers

Reply:
Good point, I'll rename them instead as they look fine as long as they aren't identical to whatever sentence predeces them.